# Sheng-Kai Hsu

**Affiliation:** Institute for Genomic Diversity, Cornell University  
**Email:** sh2246@cornell.edu  
**ORCID:** [0000-0002-6942-7163](https://orcid.org/0000-0002-6942-7163)


# Positions & Education

| Year         | Position / Degree                                                                 |
|--------------|------------------------------------------------------------------------------------|
| 2022–present | Postdoctoral research fellow, Institute for Genomic Diversity, Cornell University  |  
|              | *Advisor: Edward Buckler*                      |
| 2017–2021    | Ph.D., Vienna Graduate School of Population Genetics, Univ. of Veterinary Medicine Vienna  |
|              | *Advisor: Christian Schlötterer* |
| 2014–2015    | M.Sc., Department of Agronomy, National Taiwan University  |
|              | *Advisor: Chih-Wei Tung* |
| 2010–2014    | B.Sc., Department of Agronomy, National Taiwan University                         |


# Publications

## Peer-reviewed Publications

1. Schulz, A.J., Zhai, J., AuBuchon‐Elder, T., Andorf, C.M., El‐Walid, M.Z., Ferebee, T.H., Gilmore, E.H., Hufford, M.B., Johnson, L.C., Kellogg, E.A., & *et al.* (2025). **Fishing for a <scp>reelGene</scp>: evaluating gene models with evolution and machine learning**. *The Plant Journal*, *123*(6). https://doi.org/10.1111/tpj.70483

2. Ojeda-Rivera, J.O., Barnes, A.C., Ainsworth, E.A., Angelovici, R., Basso, B., Brindisi, L.J., Brooks, M.D., Busch, W., Buttelmann, G.L., Castellano, M.J., & *et al.* (2025). **Designing a nitrogen-efficient cold-tolerant maize for modern agricultural systems**. *The Plant Cell*, *37*(7). https://doi.org/10.1093/plcell/koaf139

3. **Hsu, S-K.**, Emmett, B.D., Haafke, A., Costa‐Neto, G., Schulz, A.J., Lepak, N., La, T., AuBuchon‐Elder, T.M., Hale, C.O., Raglin, S.S., Ojeda‐Rivera, J.O., Kent, A.D., Kellogg, E.A., Romay, M.C., & Buckler, E.S. (2025). **Contrasting rhizosphere nitrogen dynamics in Andropogoneae grasses**. *The Plant Journal*, *123*(1). https://doi.org/10.1111/tpj.70319

4. Thorhölludottir, D.A.V., **Hsu, S-K.**, Barghi, N., Mallard, F., Nolte, V., & Schlötterer, C. (2025). **Reduced Parallel Gene Expression Evolution With Increasing Genetic Divergence—A Hallmark of Polygenic Adaptation**. *Molecular Ecology*, *34*(12). https://doi.org/10.1111/mec.17803

5. Lai, W-Y., **Hsu, S-K.**, Futschik, A., & Schlötterer, C. (2025). **Pleiotropy increases parallel selection signatures during adaptation from standing genetic variation**. *eLife*, *13*. https://doi.org/10.7554/eLife.102321.3

6. **Hsu, S-K.**, Lai, W-Y., Novak, J., Lehner, F., Jakšić, A.M., Versace, E., & Schlötterer, C. (2024). **Reproductive isolation arises during laboratory adaptation to a novel hot environment**. *Genome Biology*, *25*(1). https://doi.org/10.1186/s13059-024-03285-9

7. Buchner, S., **Hsu, S-K.**, Nolte, V., Otte, K.A., & Schlötterer, C. (2023). **Effects of larval crowding on the transcriptome of <i>Drosophila simulans</i>**. *Evolutionary Applications*, *16*(10), 1671-1679. https://doi.org/10.1111/eva.13592

8. **Hsu, S-K.**, Belmouaden, C., Nolte, V., & Schlötterer, C. (2020). **Parallel gene expression evolution in natural and laboratory evolved populations**. *Molecular Ecology*, *30*(4), 884-894. https://doi.org/10.1111/mec.15649

9. Jakšić, A.M., Karner, J., Nolte, V., **Hsu, S-K.**, Barghi, N., Mallard, F., Otte, K.A., Svečnjak, L., Senti, K-A., & Schlötterer, C. (2020). **Neuronal Function and Dopamine Signaling Evolve at High Temperature in Drosophila**. *Molecular Biology and Evolution*, *37*(9), 2630-2640. https://doi.org/10.1093/molbev/msaa116

10. **Hsu, S-K.**, Jakšić, A.M., Nolte, V., Lirakis, M., Kofler, R., Barghi, N., Versace, E., & Schlötterer, C. (2020). **Rapid sex-specific adaptation to high temperature in Drosophila**. *eLife*, *9*. https://doi.org/10.7554/eLife.53237

11. **Hsu, S-K.**, Jakšić, A.M., Nolte, V., Barghi, N., Mallard, F., Otte, K.A., & Schlötterer, C. (2019). **A 24 h Age Difference Causes Twice as Much Gene Expression Divergence as 100 Generations of Adaptation to a Novel Environment**. *Genes*, *10*(2), 89. https://doi.org/10.3390/genes10020089

12. Lin, P-C., Tsai, Y-C., **Hsu, S-K.**, Ou, J-H., Liao, C-T., & Tung, C-W. (2018). **Identification of natural variants affecting chlorophyll content dynamics during rice seedling development**. *Plant Breeding*, *137*(3), 355-363. https://doi.org/10.1111/pbr.12584

13. **Hsu, S-K.**, & Tung, C-W. (2017). **RNA-Seq Analysis of Diverse Rice Genotypes to Identify the Genes Controlling Coleoptile Growth during Submerged Germination**. *Frontiers in Plant Science*, *8*. https://doi.org/10.3389/fpls.2017.00762

14. **Hsu, S-K.**, & Tung, C-W. (2015). **Genetic Mapping of Anaerobic Germination-Associated QTLs Controlling Coleoptile Elongation in Rice**. *Rice*, *8*(1). https://doi.org/10.1186/s12284-015-0072-3

## Preprints

1. Zhai, J., Gokaslan, A., **Hsu, S-K.**, Chen, S-P., Liu, Z-Y., Marroquin, E., Czech, E., Cannon, B., Berthel, A., Romay, M.C., Pennell, M., Kuleshov, V., & Buckler, E.S. (2025). **PlantCAD2: A Long-Context DNA Language Model for Cross-Species Functional Annotation in Angiosperms**. *bioRxiv.* https://doi.org/10.1101/2025.08.27.672609

2. Oren, E., Zhai, J., Rooney, T., Angelovici, R., Hale, C., Brindisi, L.J., **Hsu, S-K.**, Gault, C., Hua, J., La, T., Lepak, N., Fu, Q., Buckler, E., & Romay, C. (2025). **Grass Rhizome Proteomics Reveals Convergent Freezing-Tolerance Strategies**. *bioRxiv.* https://doi.org/10.1101/2025.05.15.654294

3. Hale, C.O., **Hsu, S-K.**, Zhai, J., Schulz, A.J., Aubuchon-Elder, T., Costa-Neto, G., Gelfond, A., El-Walid, M., Hufford, M., Kellogg, E.A., La, T., Marand, A.P., Seetharam, A.S., Scheben, A., Stitzer, M., Wrightsman, T., Romay, M.C., & Buckler, E.S. (2025). **Extensive modulation of a conserved <i>cis</i> -regulatory code across 589 grass species**. *bioRxiv.* https://doi.org/10.1101/2025.04.23.650228

4. Stitzer, M.C., Seetharam, A.S., Scheben, A., **Hsu, S-K.**, Schulz, A.J., AuBuchon-Elder, T.M., El-Walid, M., Ferebee, T.H., Hale, C.O., La, T., & *et al.* (2025). **Extensive genome evolution distinguishes maize within a stable tribe of grasses**. *bioRxiv.* https://doi.org/10.1101/2025.01.22.633974

5. Schulz, A.J., Zhai, J., AuBuchon-Elder, T., El-Walid, M., Ferebee, T.H., Gilmore, E.H., Hufford, M.B., Johnson, L.C., Kellogg, E.A., La, T., Long, E., Miller, Z.R., Romay, M.C., Seetharam, A.S., Stitzer, M.C., Wrightsman, T., Buckler, E.S., Monier, B., & **Hsu, S-K.** (2023). **Fishing for a reelGene: evaluating gene models with evolution and machine learning**. *bioRxiv.* https://doi.org/10.1101/2023.09.19.558246


# Presentations

2025 **The genetic basis of environmental adaptation in Poaceae.** *Poster presentation*, The 67th Annual Maize Genetic Meeting, March 6-9, St. Loius, MO USA  
2024 **The genetic basis of environmental adaptation in Poaceae.** *Poster presentation*, ICQG7, July 22-26, Vienna, Austria  
2024 **The phylogenetic variation in the rhizosphere nitrogen cycle of diverse grass species in the Andropogoneae.** *Oral presentation*, The 66th Annual Maize Genetic Meeting, February 29-March 3, Raleigh, NC, USA   
2024 **The phylogenetic variation in the rhizosphere nitrogen cycle of diverse grass species in the Andropogoneae.** *Oral presentation*, Plant and Animal Genome 31, January 12-17, San Diego, CA, USA   
2023 **The phylogenetic variation in the rhizosphere nitrogen cycle of diverse grass species in the Andropogoneae.** *Poster presentation*, The 65th Annual Maize Genetic Meeting, March 16-19, St. Louis, MO, USA   
2021 **Polygenic adaptation drives rapid evolution of pre- and post-mating reproductive isolation.** *Poster presentation*, The 62nd Annual Drosophila Research Conference, March 20-24, Online  
2019 **Sex-specific adaptation to high temperature in Drosophila.** *Oral presentation*, ESEB 2019, August 19-24, Turku, Finland  
2018 **Sexually antagonistic gene expression evolution in Drosophila simulans populations adapting to a novel thermal environment.** *Oral presentation*, PopGroup 51, January 3-6, Bristol, UK  
2017 **Genetics and molecular analysis of anaerobic germination in rice.** *Poster presentation*, Plant and Animal Genomes 25, January 14-18, San Diego, CA, USA   
2013 **Identification of quantitative trait loci (QTL) associated with anaerobic germination of rice (Oryza Sativa).** *Poster presentation*, The 7th International Rice Genetics Symposium, November 4-8, Manila, Philippines  




_Last updated: 2025-11-01_
